nifi-bootstrap/src/main/java/org/apache/nifi/bootstrap/util/SecureNiFiConfigUtil.java

```diff
@@ -117,6 +123,17 @@ public static void configureSecureNiFiProperties(final String nifiPropertiesFile
                 String[] subjectAlternativeNames = getSubjectAlternativeNames(nifiProperties, cmdLogger);
                 tlsConfiguration = KeyStoreUtils.createTlsConfigAndNewKeystoreTruststore(StandardTlsConfiguration
                         .fromNiFiProperties(nifiProperties), CERT_DURATION_DAYS, subjectAlternativeNames);
+                final KeyStore keyStore = KeyStoreUtils.loadKeyStore(tlsConfiguration.getKeystorePath(),
+                        tlsConfiguration.getKeystorePassword().toCharArray(), tlsConfiguration.getKeystoreType().getType());
+                final Enumeration<String> aliases = keyStore.aliases();
+                while (aliases.hasMoreElements()) {
+                    final String alias = aliases.nextElement();
+                    final Certificate certificate = keyStore.getCertificate(alias);
+                    if (certificate != null) {
+                        final byte[] sha256 = DigestUtils.getSha256Digest().digest(certificate.getEncoded());
```

Comment:
This could be streamlined to use the `sha256Hex()` method and the Hex.encodeHexString() on the following line could be removed.
```suggestion
                        final String sha256 = DigestUtils.sha256Hex(certificate.getEncoded());
```